include/zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* autogenerated using Nordic HAL utils/gen_offsets.py script */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_MISC_NORDIC_NRF_FICR_NRF54H20_H_
```

Comment:
shouldn't the enga header be deleted?

Reply:
It is deleted in cca0d2df083f9b4b8146a5ea2cb85db9e3318bb4.